### PR TITLE
Add error codes to reports

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1205608778367074/f

### Description
Add error codes to broken site reports

### Steps to test this PR
- [x] install from this branch
- [x] launch the app and load http://paco.patata
- [x] an error will occur, report broken site
- [x] verify the `epbf` contains the error as metadata

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3652** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3652" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3652?utm_source=stack-comment).

File: app/src/main/java/com/duckduckgo/app/brokensite/BrokenSiteActivity.kt [hunk 1]
```diff
@@ -165,6 +169,8 @@ class BrokenSiteActivity : DuckDuckGoActivity() {
         private const val CONSENT_MANAGED_EXTRA = "CONSENT_MANAGED_EXTRA"
         private const val CONSENT_OPT_OUT_FAILED_EXTRA = "CONSENT_OPT_OUT_FAILED_EXTRA"
         private const val CONSENT_SELF_TEST_FAILED_EXTRA = "CONSENT_SELF_TEST_FAILED_EXTRA"
+        private const val ERROR_CODES = "ERROR_CODES"
+        private const val HTTP_ERROR_CODES = "ERROR_CODES"
```
Comment (jonathanKingston) [inline app/src/main/java/com/duckduckgo/app/brokensite/BrokenSiteActivity.kt:173]:
```suggestion
        private const val HTTP_ERROR_CODES = "HTTP_ERROR_CODES"
```

**Review by marcosholgado — APPROVED** [review]
LGTM!

**Comment by jonathanKingston** [comment]
LGTM also... (I don't have perms to approve or revoke my previous comment either :') )

**Pull request merged**